shipyard-controller/db/mongodb_project_credentials_repo.go

```diff
@@ -0,0 +1,101 @@
+package db
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"time"
+
+	apimodels "github.com/keptn/go-utils/pkg/api/models"
+	"github.com/keptn/keptn/shipyard-controller/models"
+	"go.mongodb.org/mongo-driver/bson"
+)
+
+type MongoDBProjectCredentialsRepo interface {
```

Comment:
let's rename this interface to `ProjectCredentialsRepo`, as the `MongoDBRepo` is a concrete implementation of that interface